package.json

```diff
@@ -209,7 +209,8 @@
     "pg-native": "^3.0.1"
   },
   "overrides": {
-    "nx": "15.8.9"
+    "nx": "15.8.9",
+    "stream-shift": "1.0.2"
```

Comment:
We need to add a hint for Node v14 somewhere.
Node v14 automatically ships with NPM v6. But NPM v6 does not use NPM v8. Overrides came available in Node v8.3.

For CircleCI we already set the npm version for Node v14 to v8.
But locally we do not have a strategy yet.

Reply:
Maybe we can simply document to run `nvm install-latest-npm` when switching the node version with nvm?
I could not see a way to automate this e.g. with a hook from nvm.

Reply:
That makes sense. I think it's better to put this information in our contributing.md since it's related to the local development environment issue. I'll include a short note about it.

Reply:
Yes 👍

Reply:
I have added a short note.